### ISSUE TITLE
German translation 

### ISSUE THREAD

**Issue by Deflaktor** [paradigmnomad/CFDE#5]
Hi, I see that the German translation is incomplete. I want to help with it, where can I find the incomplete translation?

**Comment by Molenfeuer** [comment]
I'm on it. I wrote a letter to the maintainer of CFDE, gizzymo, and he listed me to guys who already work on it to complete the German translation. I'm waiting for an answer from both of them! Furthermore, I can keep you informed if you wish to help anyway. If the translators no longer want to do the work, we can complete the project :)

**Comment by Deflaktor** [comment]
Yeah, if some intermediate state is available it would make sense to have it in a public git repo so that anyone can contribute.
